tests/core/resource/test_events.py

```diff
@@ -424,6 +425,52 @@ def test_events_are_not_sent_if_subrequest_fails(self):
         self.assertEqual(len(self.events), 0)
 
 
+class CascadingEventsTest(BaseEventTest, unittest.TestCase):
+    subscribed = (ResourceChanged, AfterResourceChanged)
+
+    @classmethod
+    def listener(cls, event):
+        """Have the first "de Paris" event trigger a follow-up "de New York" event."""
+        cls.events.append(event)
+        if not isinstance(event, ResourceChanged):
+            return
+        if len(event.impacted_records) > 1:
+            raise ValueError("Too many events {}".format(event.impacted_records))
+        # An event without records is impossible.
+        assert len(event.impacted_records) == 1
+        if event.impacted_records[0]['new']['name'] == 'de Paris':
+            new_record = {'name': 'de New York'}
+            # Trying to match the Mushroom (i.e. UserResource) parent
+            # ID to test event grouping
+            parent_id = event.request.prefixed_userid
+            notify_resource_event(event.request, parent_id, event.payload['timestamp'], new_record,
+                                  ACTIONS.CREATE)
+
+    def test_event_can_trigger_other_event(self):
+        self.app.post_json(self.collection_url, self.body,
+                           headers=self.headers, status=201)
+        resource_changed_events = [e for e in self.events if isinstance(e, ResourceChanged)]
+        self.assertEqual(len(resource_changed_events), 2)
+        self.assertEqual(resource_changed_events[0].payload['action'], ACTIONS.CREATE.value)
+        self.assertEqual(len(resource_changed_events[0].impacted_records), 1)
```

Comment:
what's `resource_changed_events[0].impacted_records[0]['new']['name']` here?

Reply:
It should match the order of the events, so `de Paris` (which triggers the `de New York`).